tests/cross/src/test/scala/tests/pc/CompletionOverrideSuite.scala

```diff
@@ -137,7 +142,20 @@ class CompletionOverrideSuite extends BaseCompletionSuite {
       |  }
       |}
       |""".stripMargin,
-    assertSingleItem = false
+    assertSingleItem = false,
+    compat = Map(
+      "3" -> // TODO: param names should be file, attrs
```

Comment:
Actually does it work for Scala 2? At least I think it doesn't work for the code action :thinking: 

I will take a look into that one (code action) and you can follow up on this one with a new PR?

Reply:
Yeah, I'm wondering why this is working well in Scala2... I'll take a look into it, but will tackle this in another PR/issue.

Reply:
IT does work because of the completion item resolution, but it doesn't for implement all and the code action. I have a fix ready!

Reply:
Awesome! I'll take a look tomorrow

Reply:
I took a look at the PR and it's LGTM almost :+1:
Also, re-populated this test case for Scala3 https://github.com/scalameta/metals/pull/3897/commits/a484aec9e5a0a548ecc069359496a43cd842f6cd